### PR TITLE
[SP-6126] - JSON parser manifest

### PR DESCRIPTION
**Pull request by hagaishapira**

### CODE REVIEW

File: steps/slack/message/dynamic/manifest.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: stackpulse.io/v1
 kind: Step
 metadata:
-  version: 1.0.0
+  version: 1.0.1
```
Comment (eldadru) [inline steps/slack/message/dynamic/manifest.yaml:4]: Check if versioning of slack steps is supported

Comment (kostyay) [reply steps/slack/message/dynamic/manifest.yaml:4]: not yet

Comment (hagaishapira) [reply steps/slack/message/dynamic/manifest.yaml:4]: @kostyay said there is no support yet, I'll wait with the merge till then

Comment (hagaishapira) [reply steps/slack/message/dynamic/manifest.yaml:4]: @kostyay Any updates on this support

**Review by oriser — APPROVED** [review]

**Pull request merged**